pkg/operator/ceph/csi/operator_driver.go

```diff
@@ -288,7 +289,30 @@ func (r *ReconcileCSI) generateDriverSpec(clusterName string) (csiopv1.DriverSpe
 		},
 		DeployCsiAddons:  &CSIParam.EnableCSIAddonsSideCar,
 		CephFsClientType: cephfsClientType,
-	}, nil
+	}
+
+	// Apply Multus annotations if the cluster uses Multus networking
+	if cluster.Spec.Network.IsMultus() {
+		// Apply Multus to ControllerPlugin only
+		if driverSpec.ControllerPlugin.PodCommonSpec.Annotations == nil {
+			driverSpec.ControllerPlugin.PodCommonSpec.Annotations = make(map[string]string)
+		}
+		controllerObjectMeta := metav1.ObjectMeta{
+			Annotations: driverSpec.ControllerPlugin.PodCommonSpec.Annotations,
```

Comment:
Is it possible for users to add custom annotations? If not, we should have that in a follow-up PR.

Reply:
did we have this option for older driver? I don't think so.

Reply:
Oh I guess not, I must have been thinking of the custom labels